presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -37,6 +37,9 @@ including Cloudera CDH 5 and Hortonworks Data Platform (HDP).
 Many distributed storage systems including HDFS, Amazon S3, Google Cloud Storage,
 Azure Storage, and S3 compatible systems are supported.
 
+The coordinator and all workers need to have network access to the HMS, with
```

Comment:
I thought workers can do without HMS access, since they do not need metadata. is this always true? @findepi

Reply:
We found that not to be the case ...

Reply:
@phd3 think about inserts into partitioned tables. Worker needs to know where is the partition for the current row, when encountering given partition for the first time.

https://github.com/prestosql/presto/blob/ec8db2094724aeeb0bf08452268967d999bd2e40/presto-hive/src/main/java/io/prestosql/plugin/hive/HivePageSinkProvider.java#L149

Reply:
got it, thanks for the explanation.